internal/cmd/commands/dev/dev.go

```diff
@@ -378,6 +380,25 @@ func (c *Command) Run(args []string) int {
 	c.InfoKeys = append(c.InfoKeys, "worker public addr")
 	c.Info["worker public addr"] = c.Config.Worker.PublicAddr
 
+	// Check for auth token overrides
+	if c.Config.Controller.AuthTokenMaxDuration != "" {
```

Comment:
We don't actually parse a config in dev mode so this code will never actually do anything. You may want to instead add flags to control this behavior and have the flag parsing happen here. Or you can just remove this entirely for the moment if you don't want flags at this time.

Reply:
Fixed